internal/ingress/controller/template/template.go

```diff
@@ -313,7 +313,8 @@ func buildProxyPass(host string, b interface{}, loc interface{}) string {
 	}
 
 	// defProxyPass returns the default proxy_pass, just the name of the upstream
-	defProxyPass := fmt.Sprintf("proxy_pass %s://%s;", proto, upstreamName)
+	//defProxyPass := fmt.Sprintf("proxy_pass %s://%s;", proto, upstreamName)
+	defProxyPass := fmt.Sprintf("proxy_pass %s://upstream_balancer;", proto)
```

Comment:
this should be done only if the dynamic reconfiguration is enabled for the vhost/ingress